exam/src/main/java/in/testpress/exam/ui/ReviewStatsFragment.java

```diff
@@ -313,24 +316,23 @@ private void showOrHidePercentageLayout() {
     }
 
     private void showOrHideScoreLayout() {
-        if ((exam.getShowScore()) && (attempt.getScore() == null || attempt.getScore().equals("NA"))) {
-            scoreLayout.setVisibility(View.GONE);
-        } else {
+        if ((exam == null || exam.getShowScore()) && attempt.getScore() != null && !attempt.getScore().equals("NA")) {
             score.setText(attempt.getScore());
+        } else {
+            scoreLayout.setVisibility(View.GONE);
         }
     }
 
     private void showOrHidePercentileLayout() {
-        if ((exam.getShowPercentile()) &&
-                (attempt.getPercentile() == null || attempt.getPercentile().equals("NA"))) {
-            percentileLayout.setVisibility(View.GONE);
-        } else {
+        if (isExamNotNull() && exam.getShowPercentile() && attempt.getPercentile() != null && !attempt.getPercentile().equals("NA")) {
```

Comment:
`attempt.hasPercentile()`
This should check if percentile is not null and not equal to "NA". This can be directly committed to master